service/src/main/java/com/fitgoal/service/impl/NotificationServiceImpl.java

```diff
@@ -0,0 +1,65 @@
+package com.fitgoal.service.impl;
+
+import com.fitgoal.api.NotificationService;
+import com.fitgoal.api.domain.Recipient;
+import com.fitgoal.dao.domain.AuditDto;
+import com.fitgoal.api.domain.UserVerification;
+import com.fitgoal.service.AuditService;
+import com.fitgoal.service.enums.Notification;
+import com.fitgoal.service.util.MailSender;
+
+import javax.inject.Inject;
+import java.time.LocalDateTime;
+
+public class NotificationServiceImpl implements NotificationService {
+
+    private final AuditService auditService;
+    private final MailSender mailSender;
+
+    @Inject
+    public NotificationServiceImpl(AuditService auditService, MailSender mailSender) {
+        this.auditService = auditService;
+        this.mailSender = mailSender;
+    }
+
+    @Override
+    public void register(UserVerification userVerification) {
+        String link = "http://localhost:9191/verify/".concat(userVerification.getVerificationLink());
```

Comment:
`String#concat` doesn't make your code look better than just using `+` :-) So I'd recommend to not use it unless you have a special need.